pandas/core/reshape.py

```diff
@@ -1126,6 +1128,14 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
     1  0  1    0
     2  0  0    1
 
+    >>> s1 = ['a', 'b', np.nan]
```

Comment:
Not sure what you mean? See below. 
<img width="263" alt="screen shot 2017-04-06 at 5 44 43 pm" src="https://cloud.githubusercontent.com/assets/5296416/24778546/c46d6f88-1af0-11e7-8cfb-d24bf2bf92ee.png">